### PR TITLE
qemu.tests.balloon_stress:add code import and object define

### PR DESCRIPTION
**Pull request by aliang123**
balloon_stress:
1.import random
2.define object balloon_test

Signed-off-by:Aihua Liang<aliang@redhat.com>

ID:1388321

### CODE REVIEW

**Comment by aliang123** [comment]
PASS 1-smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.balloon_stress

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

File: qemu/tests/balloon_stress.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import time
 import logging
+import random
 
 from autotest.client.shared import error
 from virttest import utils_misc
 from virttest import utils_test
+from qemu.tests.balloon_check import BallooningTestWin
```
Comment (suqinhuang) [inline qemu/tests/balloon_stress.py]: from qemu.tests import balloon_check,
and then use balloon_check.BallooningTestWin will be more clear for the user

**Pull request merged**